healthcare/api-client/v1/dicom/dicomweb.py

```diff
@@ -0,0 +1,434 @@
+
+    dicomweb_path = "{}/datasets/{}/dicomStores/{}/dicomWeb/studies".format(
+        url, dataset_id, dicom_store_id
+    )
+
+    # Refine your search by appending DICOM tags to the
+    # request in the form of query parameters. This sample
+    # searches for studies containing a patient's name.
+    params = {"PatientName": "Sally Zhang"}
+
+    session = get_session()
+
+    response = session.get(dicomweb_path, params=params)
+
+    response.raise_for_status()
+
+    print("Studies found: response is {}".format(response))
+
+    # Uncomment the following lines to process the response as JSON.
+    # patients = response.json()
+    # print('Patients found matching query:')
+    # print(json.dumps(patients, indent=2))
+
+    # return patients
+
+
+# [END healthcare_dicomweb_search_studies]
+
+
+# [START healthcare_dicomweb_retrieve_instance]
+def dicomweb_retrieve_instance(
+    base_url,
+    project_id,
+    cloud_region,
+    dataset_id,
+    dicom_store_id,
+    study_uid,
+    series_uid,
+    instance_uid,
+):
+    """Handles the GET requests specified in the DICOMweb standard."""
+    url = "{}/projects/{}/locations/{}".format(base_url, project_id, cloud_region)
+
+    dicom_store_path = "{}/datasets/{}/dicomStores/{}".format(
+        url, dataset_id, dicom_store_id
+    )
+
+    dicomweb_path = "{}/dicomWeb/studies/{}/series/{}/instances/{}".format(
+        dicom_store_path, study_uid, series_uid, instance_uid
+    )
+
+    file_name = "instance.dcm"
+
+    # Make an authenticated API request
+    session = get_session()
+
+    headers = {"Accept": "application/dicom; transfer-syntax=*"}
+
+    response = session.get(dicomweb_path, headers=headers)
+
+    response.raise_for_status()
+
+    with open(file_name, "wb") as f:
+        f.write(response.content)
+        print(
+            "Retrieved DICOM instance and saved to {} in current directory".format(
+                file_name
+            )
+        )
+
+    return response
+
+
+# [END healthcare_dicomweb_retrieve_instance]
+
+
+# [START healthcare_dicomweb_retrieve_rendered]
+def dicomweb_retrieve_rendered(
+    base_url,
+    project_id,
+    cloud_region,
+    dataset_id,
+    dicom_store_id,
+    study_uid,
+    series_uid,
+    instance_uid,
+):
+    """Handles the GET requests specified in the DICOMweb standard."""
+    url = "{}/projects/{}/locations/{}".format(base_url, project_id, cloud_region)
+
+    dicom_store_path = "{}/datasets/{}/dicomStores/{}".format(
+        url, dataset_id, dicom_store_id
+    )
+
+    instance_path = "{}/dicomWeb/studies/{}/series/{}/instances/{}".format(
+        dicom_store_path, study_uid, series_uid, instance_uid
+    )
+
+    dicomweb_path = "{}/rendered".format(instance_path)
+
+    file_name = "rendered_image.png"
+
+    # Make an authenticated API request
+    session = get_session()
+
+    headers = {"Accept": "image/png"}
+
+    response = session.get(dicomweb_path, headers=headers)
+
+    response.raise_for_status()
```

Comment:
nit: These newlines don't seem neccesary
```suggestion
    headers = {"Accept": "image/png"}
    response = session.get(dicomweb_path, headers=headers)
    response.raise_for_status()
```

Reply:
Done.